docker/docker-compose.yml

```
@@ -0,0 +1,15 @@
+services:
+
+  backend:
+    image: laligaapp-backend:latest
+    restart: unless-stopped
+    networks:
+      - internal
+  
+  frontend:
+    image: laligaapp-frontend:latest
+    restart: unless-stopped
+    networks:
+      - internal
+    ports:
+      - 80:80
```

El puerto 8080 en vez de 80 evita conflictos con otros servicios.:
  ```suggestion
  services:
    backend:
      build:
        context: .
        dockerfile: docker/Dockerfile.backend
      image: laligaapp-backend:latest
      restart: unless-stopped
      networks:
        - internal
      environment:
        - APP_PORT=3005
      healthcheck:
        test: ["CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost:3005/health"]
        interval: 30s
        timeout: 3s
        retries: 3

    frontend:
      build:
        context: .
        dockerfile: docker/Dockerfile.frontend
      image: laligaapp-frontend:latest
      restart: unless-stopped
      depends_on:
        backend:
          condition: service_healthy
      networks:
        - internal
      ports:
        - "8080:80"

  networks:
    internal:
      driver: bridge
  ```